what4/src/What4/Protocol/SMTLib2.hs

```diff
@@ -1167,14 +1167,17 @@ solverMaxVersions = []
 data SolverVersionCheckError =
   UnparseableVersion Versions.ParsingError
 
-ppSolverVersionCheckError :: SolverVersionCheckError -> PP.Doc
-ppSolverVersionCheckError =
-  (PP.text "Unexpected error while checking solver version: " PP.<$$>) .
-  \case
-    UnparseableVersion parseErr -> PP.cat $ map PP.text
-      [ "Couldn't parse solver version number: "
-      , show parseErr
-      ]
+ppSolverVersionCheckError :: SolverVersionCheckError -> PP.Doc ann
+ppSolverVersionCheckError err =
+  PP.vsep
+  [ "Unexpected error while checking solver version:"
+  , case err of
+      UnparseableVersion parseErr ->
+        PP.hsep $ map PP.pretty
+        [ "Couldn't parse solver version number:"
+        , show parseErr
```

Comment:
Should this be a `pretty`, or at least a `viaShow`?

Reply:
Yes, nice catch; this one evaded my `grep` search for pretty+show combinations.